modules/users/tests/server/user-profile.server.routes.tests.js

```diff
@@ -571,6 +573,161 @@ describe('User profile CRUD tests', function () {
     });
   });
 
+  describe('Username Update', function () {
```

Comment:
I'd be in favor of moving the tests into a separate file. This may soon go out of hand.

Possible in future PR.

Reply:
Yeah, valid point!